hub/util/get_property.py

```diff
@@ -6,7 +6,7 @@
 
 def get_property(prop, project):
```

Comment:
Can this function be removed now that it is no longer used by `setup.py`? Is it used anywhere else?

Reply:
Used here: https://github.com/activeloopai/Hub/blob/release/2.0/hub/client/client.py#L5

But I don't get why this can't be just `self.version = hub.__version__` ?

Reply:
Done.

Reply:
That passes locally but fails on CI with `'Tensor version does not exist.'`... weird.